src/code/z_map_disp.c

```diff
@@ -1,109 +1,1828 @@
 #include "global.h"
+#include "gfx.h"
+#include "sys_cmpdma.h"
+#include "assets/interface/icon_item_dungeon_static/icon_item_dungeon_static.h"
+#include "assets/interface/parameter_static/parameter_static.h"
+#include "assets/objects/gameplay_keep/gameplay_keep.h"
+#include "overlays/actors/ovl_Door_Shutter/z_door_shutter.h"
+#include "overlays/kaleido_scope/ovl_kaleido_scope/z_kaleido_scope.h"
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_disp/func_80102E40.s")
+void MapDisp_DestroyMapI(PlayState* play);
+void MapDisp_InitMapI(PlayState* play);
 
-#pragma GLOBAL_ASM("asm/non_matchings/code/z_map_disp/func_80102E90.s")
+#include "gDPLoadTextureBlock_Runtime.c"
```

Comment:
```suggestion
#include "gDPLoadTextureBlock_Runtime.inc.c"
```
Got missed with the file rename.